src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/pool-list/pool-list.component.html

```diff
@@ -1,33 +1,51 @@
-<cd-table [data]="data"
-          columnMode="flex"
-          [columns]="columns"
-          identifier="name"
-          forceIdentifier="true"
-          [autoReload]="-1"
-          (fetchData)="refresh()"
-          selectionType="single"
-          (updateSelection)="updateSelection($event)"
-          [status]="tableStatus">
-  <cd-table-actions class="table-actions"
-                    [permission]="permission"
-                    [selection]="selection"
-                    [tableActions]="tableActions">
+<cd-table
+  #table
+  [data]="data"
+  columnMode="flex"
+  [columns]="columns"
+  identifier="name"
+  [searchableObjects]="true"
+  [serverSide]="true"
+  forceIdentifier="true"
+  selectionType="single"
+  [hasDetails]="true"
+  [status]="tableStatus"
+  [maxLimit]="25"
+  [autoReload]="-1"
+  (setExpandedRow)="setExpandedRow($event)"
+  (updateSelection)="updateSelection($event)"
+>
+  <cd-table-actions
+    class="table-actions"
+    [permission]="permission"
+    [selection]="selection"
+    [tableActions]="tableActions"
+  >
```

Comment:
This are unnecessary styling changes, right?

Reply:
some of them I removed some are required